examples/layeredeffects/GT.fst

```diff
@@ -122,9 +117,6 @@ let labs #i (n:int) : GTD nat i =
   then -n
   else n
 
-// GM: This fails, which I think makes sense since the effect
-//     doesn't carry any logical payload, so the assume gets lost?
-[@@expect_failure]
```

Comment:
This used to fail earlier because of the monotonicity precondition in `lift_pure_gtd`. But given that pure wps are already monotonic, we don't need that precondition.